build.gradle

```diff
@@ -636,9 +637,8 @@ afterEvaluate {
         task renameDeb(type: Copy) {
             from("$buildDir/distributions")
             into("$buildDir/distributions")
-            include archiveName
```

Comment:
for my knowledge, why remove this include and the one above?

Reply:
`archiveName` was deprecated in Gradle 7 and removed in Gradle 8: See `archiveName` in the gradle 7.6 docs here: https://docs.gradle.org/7.6/dsl/org.gradle.api.tasks.bundling.Jar.html#org.gradle.api.tasks.bundling.Jar:archiveFileName